wkcuber/wkcuber/check_equality.py

```diff
@@ -1,121 +1,38 @@
-import logging
-from argparse import ArgumentParser, Namespace
-from pathlib import Path
-from typing import Optional, Tuple
-
-import numpy as np
-from webknossos import Dataset, View
-
-from ._internal.utils import (
-    AsciiArgumentParser,
-    add_distribution_flags,
-    add_verbose_flag,
-    parse_path,
-    setup_logging,
-    setup_warnings,
-)
-from .compress import BACKUP_EXT
-
-
-def create_parser() -> ArgumentParser:
-    parser = AsciiArgumentParser()
-
-    parser.add_argument(
-        "source_path", help="Path to input WKW dataset", type=parse_path
-    )
-
-    parser.add_argument(
-        "target_path",
-        help="WKW dataset with which to compare the input dataset.",
-        type=parse_path,
-    )
-
-    parser.add_argument(
-        "--layer_name",
-        "-l",
-        help="Name of the layer to compare (if not provided, all layers are compared)",
-        default=None,
-    )
+"""This module checks equality of two different WEBKNOSSOS datasets."""
 
-    add_verbose_flag(parser)
-    add_distribution_flags(parser)
-
-    return parser
-
-
-def assert_equality_job(args: Tuple[View, View, int]) -> None:
-    source_view, target_view, _ = args
-
-    assert np.all(
-        source_view.read() == target_view.read()
-    ), f"Data differs in bounding box {source_view.bounding_box}."
-
-
-def check_equality(
-    source_path: Path, target_path: Path, args: Optional[Namespace] = None
+from pathlib import Path
+from typing import Annotated
+
+import typer
+from rich import print as rprint
+
+from webknossos import Dataset
+
+app = typer.Typer(invoke_without_command=True)
+
+
+@app.callback()
+def main(
+    source: Annotated[
+        Path,
+        typer.Argument(
+            help="Path to your first WEBKNOSSOS dataset.", show_default=False
+        ),
+    ],
+    target: Annotated[
+        Path,
+        typer.Argument(
+            help="Path to your second WEBKNOSSOS dataset.", show_default=False
+        ),
+    ],
 ) -> None:
-    logging.info(f"Comparing {source_path} with {target_path}")
+    """[Not supported yet] Check equality of two WEBKNOSSOS datasets."""
 
-    source_dataset = Dataset.open(source_path)
-    target_dataset = Dataset.open(target_path)
-
-    source_layer_names = set(source_dataset.layers.keys())
-    target_layer_names = set(target_dataset.layers.keys())
-
-    layer_names = list(source_layer_names)
-
-    if args is not None and args.layer_name is not None:
-        assert (
-            args.layer_name in source_layer_names
-        ), f"Provided layer {args.layer_name} does not exist in source dataset."
-        assert (
-            args.layer_name in target_layer_names
-        ), f"Provided layer {args.layer_name} does not exist in target dataset."
-        layer_names = [args.layer_name]
+    rprint(f"Comparing [blue]{source}[/blue] with [blue]{target}[/blue] ...")
```

Comment:
It probably makes sense to just implement this command. All the code should be found in the old check_equality module.